targets/ath79-generic

```diff
@@ -297,6 +297,9 @@ device('tp-link-tl-wdr4300-v1', 'tplink_tl-wdr4300-v1')
 device('tp-link-tl-wr1043nd-v3', 'tplink_tl-wr1043nd-v3', {
 	manifest_aliases = {
 		'tp-link-tl-wr1043n-nd-v3', -- upgrade from OpenWrt 19.07
```

Comment:
Build failing, you accidentally removed a few parentheses here.

Reply:
Thanks